contrib/plugins/todo-list/package.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
There are quite a few downsides putting the entire plugin in contrib. I don't think we're getting any dependency bumps and there is no tests that make sure that we update these examples accordingly.
Could we move these plugins into the regular plugin directory and make it an example-plugin-todo-list instead(which we don't publish ofc)? That would allow us to keep it updated with dependency bumps and ensure that we continue to have a functioning example with some unit tests (those could also be great for users to have for their own plugins).

Reply:
good point!